services/submission/src/main/java/app/coronawarn/server/services/submission/ServerApplication.java

```diff
@@ -51,7 +63,24 @@ ProtobufHttpMessageConverter protobufHttpMessageConverter() {
     return new ProtobufHttpMessageConverter();
   }
 
-  public static void main(String[] args) {
-    SpringApplication.run(ServerApplication.class);
+  @Override
+  public void setEnvironment(Environment environment) {
+    List<String> profiles = Arrays.asList(environment.getActiveProfiles());
+    if (profiles.contains("disable-ssl-server")) {
+      logger.warn("The submission service is started with endpoint TLS disabled. "
+          + "This should never be used in PRODUCTION!");
```

Comment:
Could we use a final String for the second sentence?

Reply:
Tried it, but if we go for simple String concatenation, then Sonar goes berserk. We could introduce Message formatters, but IMHO that's really not worth it. So I reverted.